verify.js

```diff
@@ -1,8 +1,19 @@
-const { readResources, listCatalog } = require('.')
+import { readResources, listCatalog } from './index.js'
 
-const assert = require('node:assert')
-const test = require('node:test')
-const { Engine, FilterFormat, FilterSet } = require('adblock-rs')
+import assert from 'node:assert'
+import crypto from 'crypto'
+import test from 'node:test'
+import { Engine, FilterFormat, FilterSet } from 'adblock-rs'
+
+const getIDFromBase64PublicKey = (key) => {
```

Comment:
I thought you said we already have this code somewhere? Can we not just export that function?

Reply:
It's deep in `brave-core-crx-packager` internals. I considered using it directly but it doesn't seem worth it to expose that and pull it in as a new dependency here just for CI purposes